### DISCUSSION TITLE
[Bug] 使用客户端请求模式时出错

### DISCUSSION THREAD

**Discussion by zhairuoxun** [lobehub/lobehub#2392] [General Question]
### 💻 Operating System

Windows

### 📦 Environment

Vercel / Zeabur / Sealos

### 🌐 Browser

Edge

### 🐛 Bug Description

打开“使用客户端请求模式”选项时，做连通性检查会出现报错
```json
{
  "error": {},
  "endpoint": "https://plugin.sm****ot.com/v1",
  "provider": "openai"
}
```
用同样的配置，不打开这个选项，就可以通过

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment]
有没有更加详细的提示？看下控制台有没有什么报错？

[a comment by an automated account is not shown]

**Comment by zhairuoxun** [comment]
> 有没有更加详细的提示？看下控制台有没有什么报错？

您好，我看到的报错信息就只有
```json
{
  "error": {},
  "endpoint": "https://plugin.sm****ot.com/v1",
  "provider": "openai"
}
```
请问您说的控制台报错在哪里查看？我找一下

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment]
![Uploading image.png…]()

在里面找到红的：

![Uploading image.png…]()

**Comment by zhairuoxun** [comment]
您好，不好意思，我看不到您发的图
![image](https://github.com/lobehub/lobe-chat/assets/71322490/14f1b5ac-3661-4352-a0bc-c0134343e52e)

**Comment by arvinxx** [comment]
<img width="348" alt="image" src="https://github.com/lobehub/lobe-chat/assets/28616219/fc2ad58c-66a1-4f92-ac03-8a39a679a420">
<img width="758" alt="image" src="https://github.com/lobehub/lobe-chat/assets/28616219/3f6dfac4-9bfc-44ff-9d69-4e28dde42564">

**Comment by zhairuoxun** [comment]
![image](https://github.com/lobehub/lobe-chat/assets/71322490/57f0e860-3c0f-4582-ac11-c35d84d20c2c)
您好，请问这个图片是否可以？

**Comment by arvinxx** [comment]
箭头所指的这两个展开看下分别截图看下？

<img width="829" alt="image" src="https://github.com/lobehub/lobe-chat/assets/28616219/a37191cc-7b69-49a9-9b09-feaf5f42fd61">

**Comment by zhairuoxun** [comment]
“响应标头”原本就是展开的状态，里面什么都没有，”响应“ 部分展开后是这样的
![image](https://github.com/lobehub/lobe-chat/assets/71322490/a78da106-2c51-493f-af3d-f7df019cfa28)

**Comment by arvinxx** [comment]
@zhairuoxun 那你点下这

<img width="612" alt="image" src="https://github.com/lobehub/lobe-chat/assets/28616219/f31483aa-6d50-4f0c-be49-fbc85ce66e14">

然后再截个完整的图，我看下报错的响应码

**Comment by zhairuoxun** [comment]
![image](https://github.com/lobehub/lobe-chat/assets/71322490/8e2188d9-a118-4b98-9365-d20716d57588)
![image](https://github.com/lobehub/lobe-chat/assets/71322490/76090e10-e33a-4eab-affe-aebe59c6e52b)
![image](https://github.com/lobehub/lobe-chat/assets/71322490/fabd28c9-d9a8-4eee-b4fe-9c34d65a11a8)
![image](https://github.com/lobehub/lobe-chat/assets/71322490/31938776-4524-47d7-b037-03135dca6b55)

您好,这是完整的截图, 余下两个标签页都写着"无法加载响应数据". 我是在Edge浏览器遇到这个问题的, 换成Chrome和Firefox浏览器以后问题仍然存在

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment]
@zhairuoxun 我感觉可能是你的这个服务商不支持跨域请求导致的，能去问问看这个服务商的接口是否支持跨域不？

**Reply by zhairuoxun** [reply]
服务商表示他们是支持跨域的，他们也会测试一下自己的接口行不行，不知道会不会有结果

[a comment by an automated account is not shown]
